.github/workflows/generate-summary.yaml

```diff
@@ -454,23 +454,23 @@ jobs:
           zip -r previous_logs.zip previous_logs
 
       - name: Upload compare summaries
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: ${{ inputs.prefix }}${{ steps.bisection-hash.outputs.orig_gcchash }}-regenerated-summaries
           path: |
             riscv-gnu-toolchain/summaries.zip
           retention-days: 90
 
       - name: Upload current log failures
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: ${{ inputs.prefix }}${{ steps.bisection-hash.outputs.orig_gcchash }}-regenerated-current-logs
           path: |
             riscv-gnu-toolchain/current_logs.zip
           retention-days: 90
 
       - name: Upload previous log failures
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: ${{ inputs.prefix }}${{ steps.bisection-hash.outputs.orig_gcchash }}-regenerated-previous-logs
           path: |
```

Comment:
may also want to update the JasonEtco/create-an-issue@v2 in regenerate summary https://github.com/patrick-rivos/gcc-postcommit-ci/pull/1006/files#diff-212c3469e283d0c467d86f01ac4ab899a704631cdf26e87daf5d75bb9abcf679R509

Reply:
JasonEtco/create-an-issue@v3 does not seem to exist:
https://github.com/JasonEtco/create-an-issue